main.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/ingress-exporter/flag"
+	"github.com/giantswarm/ingress-exporter/server"
+	"github.com/giantswarm/ingress-exporter/service"
```

Comment:
It would be grand to move the project imports down in their own block. See https://github.com/giantswarm/fmt/blob/master/go/imports.md.